bundles/org.openhab.binding.bondhome/src/main/java/org/openhab/binding/bondhome/internal/handler/BondDeviceHandler.java

```diff
@@ -0,0 +1,822 @@
+                            command == OnOffType.ON ? BondDeviceAction.StartDimmer : BondDeviceAction.Stop, null);
+                    updateState(CHANNEL_STOP, OnOffType.ON);
+                    // Unset in 30 seconds when this times out
+                    scheduler.schedule(() -> {
+                        logger.trace("Fan up light dimmer change command run for 30s");
+                        updateState(CHANNEL_STOP, OnOffType.OFF);
+                        updateState(CHANNEL_UP_LIGHT_START_STOP, OnOffType.ON);
+                    }, 30, TimeUnit.SECONDS);
+                    break;
+
+                case CHANNEL_UP_LIGHT_DIRECTIONAL_INC:
+                case CHANNEL_UP_LIGHT_DIRECTIONAL_DECR:
+                    // TODO(SRGDamia1): Command format not documented by Bond for up light directional brightness
+                    logger.info("Bi-direction brightness control for up-lights not yet enabled!");
+                    break;
+
+                case CHANNEL_DOWN_LIGHT_ENABLE:
+                    api.executeDeviceAction(config.deviceId, command == OnOffType.ON ? BondDeviceAction.TurnDownLightOn
+                            : BondDeviceAction.TurnDownLightOff, null);
+                    break;
+
+                case CHANNEL_DOWN_LIGHT_STATE:
+                    // To turn on the down light, we first have to enable it and then turn on the lights
+                    api.executeDeviceAction(config.deviceId, BondDeviceAction.TurnDownLightOn, null);
+                    if (command == OnOffType.ON) {
+                        api.executeDeviceAction(config.deviceId, BondDeviceAction.TurnLightOn, null);
+                    } else {
+                        api.executeDeviceAction(config.deviceId, BondDeviceAction.TurnLightOff, null);
+                    }
+                    break;
+
+                case CHANNEL_DOWN_LIGHT_BRIGHTNESS:
+                    if (command instanceof PercentType) {
+                        PercentType pctCommand = (PercentType) command;
+                        int value = pctCommand.intValue();
+                        logger.trace("Fan down light brightness command with value of {}", value);
+                        api.executeDeviceAction(config.deviceId, BondDeviceAction.SetDownLightBrightness, value);
+                    } else if (command instanceof IncreaseDecreaseType) {
+                        logger.trace("Fan down light brightness increase/decrease command");
+                        api.executeDeviceAction(config.deviceId,
+                                ((IncreaseDecreaseType) command == IncreaseDecreaseType.INCREASE
+                                        ? BondDeviceAction.IncreaseDownLightBrightness
+                                        : BondDeviceAction.DecreaseDownLightBrightness),
+                                null);
+                    } else {
+                        logger.info("Unsupported command on fan down light brightness channel");
```

Comment:
Logging to info should be used rarely. This could be debug. Same for below.

Reply:
done